alembic/latiss/versions/99ff646cd4b0_update_the_views.py

```diff
@@ -0,0 +1,152 @@
+    op.execute("ALTER TABLE cdb_latiss.ccdvisit1 RENAME COLUMN ccdexposure_id TO ccdvisit_id")
+    op.execute("ALTER TABLE cdb_latiss.ccdvisit1 RENAME COLUMN exposure_id TO visit_id")
+    op.execute("ALTER TABLE cdb_latiss.visit1 RENAME COLUMN exposure_id TO visit_id")
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table_comment(
+        "visit1_quicklook",
+        existing_comment="Information from Summit Rapid Analysis about each visit in visit system 1 (visit per exposure); describes processed visit images (or calibrated exposures)",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "exposure_quicklook",
+        existing_comment="Information from Summit Rapid Analysis about each raw image taken",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "exposure_flexdata_schema",
+        existing_comment="Key names and value types used in the exposure_flexdata table",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "exposure_flexdata",
+        existing_comment="Flexible key/value metadata about each raw image taken; used for development and engineering purposes",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "exposure",
+        existing_comment="Information from observatory systems about each raw image taken",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "ccdvisit1_quicklook",
+        existing_comment="Information from Summit Rapid Analysis about each detector (CCD) within each visit in visit system 1 (visit per exposure); describes detectors in processed visit images (or calibrated exposures); consistent with other multi-detector instruments",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "ccdexposure_flexdata_schema",
+        existing_comment="Key names and value types used in the ccdexposure_flexdata table",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "ccdexposure_flexdata",
+        existing_comment="Flexible key/value metadata about each detector (CCD) within each raw image taken; used for development and engineering purposes; consistent with other multi-detector instruments",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "ccdexposure_camera",
+        existing_comment="Information from the Camera Control System about each detector (CCD) within each raw image taken; consistent with other multi-detector instruments",
+        schema="cdb_latiss",
+    )
+    op.drop_table_comment(
+        "ccdexposure",
+        existing_comment="Information from observatory systems about each detector (CCD) within each raw image taken; consistent with other multi-detector instruments",
+        schema="cdb_latiss",
+    )
+    op.execute("DROP VIEW cdb_latiss.ccdvisit1")
```

Comment:
These don't really have any effect.  To restore the views to the prior state, you would need to go back to before the underlying tables were updated.  But that's not really what we want either, so I'd vote for just removing these from `downgrade` here and in the LSSTCam migration.